config/environment.js

```diff
@@ -4,6 +4,7 @@ module.exports = function(environment) {
   var ENV = {
     isDevknox: false,
     isAppknox: false,
+    devknoxPrice: 9,  // This should also change in `mycroft/settings.py`
```

Comment:
@subho007 For time being. We will have a config endpoint. There is an Issue there.